index.html

```diff
@@ -12328,7 +12442,7 @@ <h3>States and Properties</h3>
 			<p>If a <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> property contains an unknown or disallowed value, the user agent SHOULD expose to platform <a>accessibility APIs</a> as follows:</p>
```

Comment:
Change "property contains"
to "state or property contains".

Reply:
This line has been removed